movedata/move.go

```diff
@@ -62,7 +83,9 @@ func (mover *Mover) MoveData() error {
 	}
 	defer elasticBulk.Close(context.Background())
 
-	log.Printf("going to execute query %s whit param %s", mover.queryConf.Query, lastDate)
+	columnsMap, jsonColsMap := getColumnsConfiguration(mover.queryConf)
```

Comment:
we don't need parse the configuration every 5seconds, you can call this method in the Mover initialization (and instead of returning two maps, you can directly store the maps in the struct state)

Reply:
still need to do this

Reply:
Done

Reply:
Also this is not realy what I was thinking, but we can do in a separate PR:

We need to modify the `Mover struct `adding c`olumnsMap, jsonColsMap`